recipe/conda_build_config.yaml

```diff
@@ -2,3 +2,4 @@ pytorch:
   - 1.8.*
   - 1.9.*
   - 1.10.*
+  - 1.11.*
```

Comment:
Why are we still building for so many pytorch versions here (and also missing the last recent two)...?

Reply:
This is detetectrons officially supported versions of the last release. let's build for the other two and get rid of the matrix completely with the pytorch113 migration